src/lib.rs

```diff
@@ -21,13 +21,23 @@ pub struct PanCamPlugin;
 #[derive(Debug, Clone, Copy, SystemSet, PartialEq, Eq, Hash)]
 pub struct PanCamSystemSet;
 
+/// Which keys move the camera in particular directions for keyboard movement.
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Reflect)]
+pub struct DirectionKeys {
+    up: Vec<KeyCode>,
+    down: Vec<KeyCode>,
+    left: Vec<KeyCode>,
+    right: Vec<KeyCode>,
+}
```

Comment:
Should we add some constants here for convenience? `NONE`, `ARROWS`, `WASD`, `ARROWS_AND_WASD`?

Reply:
I had to add these with methods (other than NONE), because allocations are not allowed in constants.